### ISSUE TITLE
Enable embedded plots in html

### ISSUE THREAD

**Issue by lexnederbragt** [doconce/doconce#189]
For `ipynb` and `pdf` output, compiled with `--execute`, a `!bc pycod`. block that results in a plot has the plot embedded in the output.
For `html` output, there is instead a sentence stating `PDF cannot be displayed in your browser. See .doconce_figure_cache/<long-id>.pdf`

Could a solution be to generate a png file from the PDF and embed that in html? Doconce has functionality for creating png, it uses ImageMagick for that. One application is when for `ipynb` when only a `pdf` version is available.

Example: compiling `src/exercise1.do.txt` in my doconce demo using `doconce format html exercise1.do.txt --execute`.

**Comment by aless80** [comment]
This is browser dependent. What's the browser, Safari? It works on mine (Firefox and Chromium for linux). 

The current output uses the `<object>` tag eg:
```
<object width="500" height="375" type="application/pdf" data=".doconce_figure_cache/2a8ec051-ed4e-4510-a66b-663806d0d3a0.pdf">
<p>PDF cannot be displayed in your browser. See .doconce_figure_cache/<long hash>.pdf</p>
</object>
```
Here's a discussion on how to embed pdfs: [stackoverflow](https://stackoverflow.com/questions/291813/recommended-way-to-embed-pdf-in-html). Maybe it would be sufficient to use both `<object>` and `<embed>` (the latter fails in my browsers but might work on yours). So: can you try to replace `<object>` with `<embed>` on that image and try both on all your browsers? 

Then we can consider to use formats other than pdf.

**Comment by lexnederbragt** [comment]
Darn, the problem was on my end: after compilation, I move the resulting html file to the `pub` folder, but not the `doconce_figure_cache/<long hash>.pdf` file, causing the figure not to e displayed. Not moving the html file results in a page with the PDF embedded.

I did do a quick test replacing `<object>` with `<embed>` but it did not work, I got a gray placeholder instead of an embedded PDF.

Closing as ' no-need-to-fix'.
